app/src/components/Modals/add-user-modal.tsx

```diff
@@ -171,23 +172,6 @@ const AddUserModal: FC<AddUserModalProps> = ({
                   label={t("email")}
                   register={register}
                 />
-
-                <FormSelectInput
-                  label={t("role")}
-                  value={inputValue}
-                  register={register}
-                  error={errors.role}
-                  id="role"
-                  onInputChange={onInputChange}
-                />
```

Comment:
This should only be shown for admins imo? Fine to comment out for now but we could also keep it and only conditionally render it when the current user is an admin?